src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellPageRendererTracker.cs

```diff
@@ -461,7 +462,13 @@ public override CGRect Frame
 
 			public override void WillMoveToSuperview(UIView newSuper)
 			{
-				if (newSuper != null)
+				UpdateFrame(newSuper);
+				base.WillMoveToSuperview(newSuper);
+			}
+
+			void UpdateFrame(UIView newSuper)
+			{
+				if (newSuper is not null && newSuper.Bounds != CGRect.Empty)
```

Comment:
This by itself causes the measure of the height to differ between iOS 16 and pre iOS 16

This code would set the Height property to the superview height, which would then cause the `Arrange` in the base class to get called with the value that `Height` is set to. 

Because `Height` is no longer being set, iOS 16 uses the measured height where as pre iOS 15 uses the Super.Bounds.Height

I reworked a lot of this code [here](https://github.com/dotnet/maui/pull/12888/files#diff-300230717000d3e80cfbbf5dad91996f86e73ebf13980f00423285ad11e85447) and from what I can tell in .NET 8 everything is now consistent with ios16 and pre ios16. 

I debated just leaving this code as is because something showing seems better than nothing but I'm worried about the TitleView measuring/arranging higher once users move from .net7 to .net8

Given the following `XAML` here are the different results

```XAML
 <Shell.TitleView>
        <Grid Background="Purple">
            <Grid.RowDefinitions>
                <RowDefinition Height="*"></RowDefinition>
                <RowDefinition Height="Auto"></RowDefinition>

            </Grid.RowDefinitions>
            <Label Text="I should exist!" Background="blue" />
            <Label Grid.Row="1" Text="I should exist!" Background="Yellow" />
        </Grid>
    </Shell.TitleView>
```

This is how it currently shows on pre iOS 16

![image](https://user-images.githubusercontent.com/5375137/229611758-86afb660-0845-4bc8-8b01-09b9aba65da3.png)

This is how it shows with this PR on iOS 16

![image](https://user-images.githubusercontent.com/5375137/229612072-830784e4-96e9-462f-9dd6-8ff39d106d9a.png)

In .NET8 we're making an additional call to UpdateFrame inside `LayoutSubviews` in order to update all those values. Once we do that then we get the following results on iOS 16 compared to iOS 13. 

![image](https://user-images.githubusercontent.com/5375137/229612648-a927bbc1-4ef3-41a6-9bb9-b1898578ab47.png)


I've pushed up the extra call to `UpdateFrame` because I think it's important if we backport this that the measure behavior stays consistent.